src/components/AnonymousChat/BaseChannelItem.tsx

```diff
@@ -81,7 +69,7 @@ const BaseChannelItem: (props: BaseChannelItemProps) => React.ReactElement = ({
             unreadCount={unreadCount}
             isMe={isMe}
             hasFollowButton={hasFollowButton}
-            isFollowing={isFollowing}
+            isFollowing={showFollowingButton}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isFollowing` prop has been replaced with `showFollowingButton`. This change is consistent with the previous changes, moving the responsibility of determining whether a user is followed to the parent component or custom hook. It's important to ensure that the `showFollowingButton` prop is being correctly set in the parent component or hook.
```diff
- isFollowing={isFollowing}
+ isFollowing={showFollowingButton}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->